src/Altinn.Broker/Controllers/FileController.cs

```diff
@@ -1,105 +1,250 @@
-using System.Diagnostics.CodeAnalysis;
-
-using Microsoft.AspNetCore.Mvc;
-
-using Altinn.Broker.Models;
+using Altinn.Broker.Core.Domain.Enums;
 using Altinn.Broker.Core.Models;
+using Altinn.Broker.Core.Repositories;
 using Altinn.Broker.Mappers;
-using Altinn.Broker.Core.Services.Interfaces;
+using Altinn.Broker.Models;
 using Altinn.Broker.Persistence;
 
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
 namespace Altinn.Broker.Controllers
```

Comment:
There seems to be missing a method equivalent to HasAvailableFiles (was HasAvailableShipments), consider implementing it.

Reply:
I will add a corollary to #165 for this